src/main/java/uk/gov/ons/census/fwmt/outcomeservice/config/RedisConfig.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.ons.census.fwmt.outcomeservice.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.RedisStandaloneConfiguration;
+import org.springframework.data.redis.connection.jedis.JedisClientConfiguration;
+import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.serializer.GenericToStringSerializer;
+import org.springframework.data.redis.serializer.JdkSerializationRedisSerializer;
+import org.springframework.data.redis.serializer.StringRedisSerializer;
+
+@Configuration
+public class RedisConfig {
+
+  private String redisHostname;
+  private int redisPort;
+
+  public RedisConfig(@Value("${redis.host}") String redisHostname,
+      @Value("${redis.port}") int redisPort) {
+    this.redisHostname = "localhost";
+    this.redisPort = redisPort;
+  }
+
```

Comment:
do you need to connect to a certain database here ?

Reply:
That's done within the jedisConnectionFactory method of this class.